principles/security-principles.md

```diff
@@ -2,8 +2,31 @@
 layout: default
 title:  "Security Principles"
 ---
+# Penetration Testing Process
+When a project is close to completion delivery managers will need to liase with the security team to arrange a penetration test. 
 
-# Security
+DM's will need to fill out the scoping document which can be found at the following links:
+1. [Web application assessments](https://github.com/SkillsFundingAgency/das-technical-guidance/blob/security-principles-update/principles/scoping_documents/Web_App_Scoping_Questionnaire%20(3).docx)
```

Comment:
It would be good to store these centrally in a repository with restricted access. Ben and Rob T did set something up some time ago for this and we used to so that the 3 of us could go through security vulnerabilities periodically. 
Do you have an area like this available? Would be good to get back into that process.

Reply:
We are storing them in the security section of SharePoint

Reply:
Also we have a spreadsheet we update after each pen test which has details of issues discovered across all pen tests done in the past year or so

Reply:
Can I access those (not on DfE kit?). I do have DfE Kit but I don't use it much.